From 60d0563f35bf41df01b8a807d7058a973b42b870 Mon Sep 17 00:00:00 2001
From: Andrej Gantvorg <andrej.gantvorg@wago.com>
Date: Wed, 31 Oct 2018 17:11:40 +0100
Subject: [PATCH 03/10] adds wago changelog.

---
 CHANGELOG.WAGO | 7 +++++++
 1 file changed, 7 insertions(+)
 create mode 100644 CHANGELOG.WAGO

diff --git a/CHANGELOG.WAGO b/CHANGELOG.WAGO
new file mode 100644
index 0000000..8a6b9c2
--- /dev/null
+++ b/CHANGELOG.WAGO
@@ -0,0 +1,7 @@
+2017.11.1: 
+    - bugfix release
+    - 2 upstream patches backported to 2017.11.0 by WAGO
+    - makes install_archive preserve owner/group info
+    - fixes ptxdist print behaviour when printing multiple variables at once
+    - see wago-patches/*.patch for details
+        
-- 
2.7.4

